### PR TITLE
Colours

### PR DESCRIPTION
**Pull request by KittenPrincess1**
Standardised the colours across all different colour schemes, got rid of excess code that made it difficult to implement new colour schemes, and added a new black and white colour scheme

### CODE REVIEW

**Comment by KittenPrincess1** [comment]
I believe it is ready to merge, so long as it doesn't cause issues with any of the other code

File: src/components/DataStructures/Array/Array1DRenderer/Array1DRenderer.module.scss
Comment (Sraccity) [inline src/components/DataStructures/Array/Array1DRenderer/Array1DRenderer.module.scss:1]: Approved: Formatting Changes only

File: src/styles/global.scss
Comment (Sraccity) [inline src/styles/global.scss:1]: Notes: Main changes here

**Review by Sraccity — APPROVED** [review]
Kudos: Seems proper

- [x] Did local testing work?

Notes for future ref: Changes in Renderer mainly. look at 
-  global.scss
-  Renderer Changes mainly

**Comment by Sraccity** [comment]
- [x] Added Black and white colour scheme
- [x] Standardised colours across all colour schemes
- [x] Commented what each renderer is responsible for
- [x] Fixed some inconsistencies

---------

Co-authored-by: @KittenPrincess1 by HumanGPT  <110475564+MelanieH7@users.noreply.github.com>

**Comment by Sraccity** [comment]
@KittenPrincess1 Congrats on your first PR

**Pull request merged**